integration/shareddata/scalars.go

```diff
@@ -28,7 +28,10 @@ const (
 	doubleBig     = float64(1 << 61)   // 2305843009213693952.0: some number larger than safe integer (doubleBig+1 == doubleBig)
 	longBig       = int64(1 << 61)     // 2305843009213693952:   same as doubleBig but integer
 
-	// TODO https://github.com/FerretDB/FerretDB/issues/2321
+	doubleMaxOverflow = float64(9223372036854776833)  // double value that is larger than MaxInt64
+	doubleMinOverflow = float64(-9223372036854776833) // double value that is smaller than MinInt64
+
+	doubleMaxVerge = 1.79769e+307 // double value that is close to the MaxFloat64 (1.79769e+308)
```

Comment:
Could you also add non scientific notation in a comment?

What is the special thing about constant `1.79769`? I know it's the same constant as `MaxFloat64`. But does this mean that adding certain number will make it overflow or?

Reply:
Do we want to represent `1.79769e+307` in standard notation? It will be huge comment...

> What is the special thing about constant 1.79769? I know it's the same constant as MaxFloat64.

It's not, this one is on a verge (`+307`) and `MaxFloat64` has (`+308`)

Reply:
The difference between 1.79769e+308 and 1.79769e+307 is 1.617921e+308; I would not call those values "close".

Could it be `math.MaxFloat64 - 1.0`?

Reply:
If tests work with `math.MaxFloat64 - 1.0`, calling that `verge` makes more sense to me :hugs: